src/Orleans/Providers/IStorageProvider.cs

```diff
@@ -70,7 +70,7 @@ public interface IStorageProvider : IProvider
     /// <summary>
     /// Interface to be optionally implemented by storage providers to return richer exception details.
     /// </summary>
-    public interface IExceptionDecoder
+    public interface IStorageExceptionDecoder
```

Comment:
It used to be 3 min. I just put back the value we had before. Reviewing those storage related timeouts is probably unrelated to this whole "eliminate hard Azure dependency PR".

Anyway, I shortened it to 1 min.